src/test/java/com/wikia/webdriver/common/core/drivers/Browsers/HtmlUnitBrowser.java

```diff
@@ -0,0 +1,17 @@
+package com.wikia.webdriver.common.core.drivers.Browsers;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00120)